client/my-sites/sidebar/sidebar.jsx

```diff
@@ -641,16 +586,40 @@ function mapStateToProps( state ) {
 	const currentUser = getCurrentUser( state );
 	const selectedSiteId = getSelectedSiteId( state );
 	const isSingleSite = !! selectedSiteId || currentUser.site_count === 1;
-	const singleSiteId = selectedSiteId || ( isSingleSite && getPrimarySiteId( state ) ) || null;
+	const siteId = selectedSiteId || ( isSingleSite && getPrimarySiteId( state ) ) || null;
+	const site = getSite( state, siteId );
+
+	const isJetpack = isJetpackSite( state, siteId );
+	// FIXME: Fun with Boolean algebra :-)
+	const isSharingEnabledOnJetpackSite = ! (
+		! isJetpackModuleActive( state, siteId, 'publicize' ) &&
+		( ! isJetpackModuleActive( state, siteId, 'sharedaddy' ) || isJetpackMinimumVersion( state, siteId, '3.4-dev' ) )
+	);
+	// FIXME: Turn into dedicated selector
+	const canManagePlugins = !! getSites( state ).some( ( s ) => (
+		( s.capabilities && s.capabilities.manage_options )
+	) );
```

Comment:
Feel free to cherry-pick (or nuke, since it follows the weird `selectedOrAll` approach of `sites-list`) of https://github.com/Automattic/wp-calypso/pull/13094/commits/d8b385b01557c7a2c387fc2cd79bcf3c301194d5